pkg/apis/kibana/v1/name_test.go

```diff
@@ -31,3 +33,27 @@ func TestHTTPService(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigSecret(t *testing.T) {
+	type args struct {
+		kb Kibana
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "sample",
+			args: args{kb: Kibana{ObjectMeta: metav1.ObjectMeta{Name: "sample"}}},
+			want: "sample-kb-config",
+		},
```

Comment:
Nit: Not sure this needs a table driven test with just one case? Maybe reactor to one table that tests all the namer functions, where the function calle is part of the test defintion?

Reply:
Updated to test all with a single test.